appveyor.yml

```diff
@@ -37,7 +37,7 @@ install:
   - "python -c \"import struct; print(struct.calcsize('P') * 8)\""
 
   # Install the build and runtime dependencies of the project.
-  - "%CMD_IN_ENV% pip install -v %WHEELHOUSE% -r tools/appveyor/requirements.txt"
+  - "%CMD_IN_ENV% pip install --trusted-host 28daf2247a33ed269873-7b1aad3fab3cc330e1fd9d109892382a.r6.cf2.rackcdn.com -r tools/appveyor/requirements.txt"
```

Comment:
Maybe try taking this addition out?  Otherwise, it must have to do with changes made in `setup.py` or `requirements.txt` (but I'm confused at this point).